2_Operations/work-process.md

```diff
@@ -1,65 +1,52 @@
- * The Tuesday, two weeks later, marks the end of the sprint with a [sprint retrospective](scrum.md#Sprint_Retrospective).
+ * A Wednesday is the start of a sprint with sprint planning.
+ * The Tuesday, two weeks later, marks the end of the sprint with a sprint retrospective.
  * Then Wednesday, the next day, is usually the start of the next sprint.
 
+The last Monday morning of the sprint everyone should open up the Scrum board and ask themselves: "How can I help close whatever is still opened?". Repeat the same after lunch and on Tuesday morning.
 
-## Creating issues and user stories
-> *Main document: [Guide for user stories](user-stories.md)*
+For each new sprint, the Scrum master will create an operations sprint release that is used to store the user story screencast demos. The template for this release is copied from [RELEASE_TEMPLATE.md](https://github.com/niteoweb/operations/raw/master/.github/RELEASE_TEMPLATE.md) in the operations repo.
 
-Undefined issues are created in various repositories to track known problems. Issues that are created by Support are usually reactionary so stay in project-related repositories and are not included in sprints.
+## Scrum
 
-When undefined issues from various repositories need to be moved into a sprint, a well-defined user story needs to be created for that issue. This can be done by the author of the issue, Scrum master or product owner.
+Our work process is based on [Scrum](https://en.wikipedia.org/wiki/Scrum_(software_development)). Read in detail about [our Scrum process](scrum.md) and writing [User Stories](user-stories.md).
 
-To create a new user story, open an issue in [operations repository](https://github.com/niteoweb/operations/issues) and use the supplied template. There are further details on writing the US in the [user stories](user-stories.md) document.
+ * Sprint length is two weeks.
+ * Story points commitment: 10 for each full-time member on the sprint. The total team commitment has at least 25% reserved for bugfixes and cleanup.
+ * Daily standup meeting times: 10:00 CE(S)T for Easy Blog Networks and, 11:00 CE(S)T for WooCart
 
-## Pull requests and commits
+## Quarterly Planning
 
-> *Main article: [Git guide](../apps-we-use/git.md)*
+TODO
 
-While working on user stories we often need to change code or text that we keep in GitHub. These changes are done with [git commits](https://help.github.com/articles/github-glossary/#commit) and are presented to others for review as [Pull Requests](https://help.github.com/articles/about-pull-requests/).
+## Alignment Meeting
 
-To ensure that your commits and Pull Requests contain everything necessary for a successful review, read through the [Git](../apps-we-use/git.md) guide for the best practices.
+15 minutes after the sprint retrospective we hold another meeting. Attendance on this one is optional, only those interested in the general direction the company is taking normally attend. That said, every Nitean is welcome to participate.
```

Comment:
Since we now have two retrospectives, this should probably be scheduled at 12 CE(S)T.

Reply:
Alternatively, we could have one alignment for EBN and one for WooCart, each fro their own part of the OKRs. 

I'd still to do it together though ... maybe we need to shuffle people between projects if one is falling behind ...

Reply:
This change will be implemented after 26th September.